client-src/index.js

```diff
@@ -46,16 +46,40 @@ const options = {
 };
 const parsedResourceQuery = parseURL(__resourceQuery);
 
+const enabledFeatures = [];
```

Comment:
In theory we will have a memory problem here, because we will put more and more, I think will:
1. create general log function
2. create `enabledFeatures` object and set true/false
3. pass `enabledFeatures` to general log function

And it will allow to us do better logging:
```
[webpack-dev-server] Server started: Hot Module Replacement enabled, Live Reloading enabled, Overlay disabled.
```

I.e. we will output `enabled` and `disabled`, so developers can see enabled/disabled features

Reply:
@alexander-akait Done, sorry for the delay.